openmicroscopy/bioformats
# Roi handler update rebase

This is PR #1442  rebased onto dev_5_0.

To test :

1) Add bioformats_package.jar to your ImageJ plugins folder
2) Open any Image in ImageJ, draw roi's and add them to the ROI Manager.
3) Try exporting images using Plugins-->Bio-Formats-->Bio-Formats Exporter.

This should write the ROI's from the ROI manager to your ome-tiff.

To check :

Using Command line tools :

---. /showinf -nopix -omexml /Users/bramalingam/Desktop/ImageJTest/finalTest.ome.tif

This should show all the ROI's , within "< ROI >" " < /ROI >"

Using Image J : 

Import the Image using Bio-formats Importer (select checkbox "Display ROI's" while import) and check if the ROI Manager is populated with the ROI's (that were drawn earlier).
